base/applications/mscutils/eventvwr/lang/zh-CN.rc

```diff
@@ -107,9 +107,9 @@ BEGIN
     LTEXT "描述(&D)：", IDC_STATIC, 8, 65, 39, 8
     CONTROL "", IDC_EVENTTEXTEDIT, RICHEDIT_CLASS, ES_MULTILINE | ES_AUTOVSCROLL | ES_READONLY | WS_VSCROLL | WS_BORDER, 8, 76, 250, 60
 
-    LTEXT "数据(&A)", IDC_DETAILS_STATIC, 8, 140, 20, 8
-    CONTROL "字节(&B)", IDC_BYTESRADIO, "Button", BS_AUTORADIOBUTTON, 39, 140, 34, 8
-    CONTROL "字(&W)", IDC_WORDRADIO, "Button", BS_AUTORADIOBUTTON, 77, 140, 34, 8
+    LTEXT "数据(&A)：", IDC_DETAILS_STATIC, 8, 140, 25, 8
```

Comment:
The static was too small?

Reply:
for en-US the size is 20. But for zh-**TW** the number was increased to 23 already. Even before I added the Chinese ":". So at some point somebody realized it being too narrow already. So I played safe here and used 25 for all 3 Chinese due to the added characters requirements.
For the record: There are also some non-US and non-Chinese languages that already used a bigger value here than the 20. It is something that is groomed by language: We do have to make sure that 8 + 25 is < start-x-position 39 of the first RADIOBUTTON. Some languages even moved the first radio-button to the right. I took care to preserve such existing language-extrawursts when I did my changes.